incubator/kubirds-powerapi-reactor/values.yaml

```diff
@@ -0,0 +1,25 @@
+---
+
+# User Configuration
+
+labels: {}
+unitSelector: {}
+
+triggers:
+  success: yes
+  failure: yes
+  fixed: no
+  regression: no
+
+# Internal Configuration (can be overidden)
+
+image:
+  pullPolicy: IfNotPresent
+  args: >
```

Comment:
You're reading `INPUT` and `OUTPUT` from the environment, this is not a best practice I think.

I was thinking of something more like:

```yaml
inputs:
  - type: csv
    dir: /var/log/sensors
outputs:
  - type: csv
    name: power
    module: PowerReport
    dir: /var/log/output
```

Assuming `-n` means `name` and `-m` means `module`.
Then in the reactor's template, you use a `{{ range }}` loop to generate the command.